app/build.gradle

```diff
@@ -137,10 +135,10 @@ dependencies {
         implementation "org.jetbrains.anko:anko-$name:$ankoVersion"
     }
     //Firebase
-    implementation 'com.google.firebase:firebase-core:16.0.8'
-    implementation 'com.google.firebase:firebase-messaging:17.4.0'
-    implementation 'com.google.firebase:firebase-invites:16.1.1'
-    implementation 'com.crashlytics.sdk.android:crashlytics:2.9.9'
+    implementation 'com.google.firebase:firebase-core:16.0.9'
```

Comment:
Don't do any changes in the dependencies

Reply:
sorry I should have removed them already

Reply:
sorry, I should have removed them already.